frontend/pages/_document.js

```diff
@@ -0,0 +1,68 @@
+import Document, { Head, Main, NextScript } from 'next/document';
+import Helmet from 'react-helmet';
+
+// from https://github.com/zeit/next.js/edit/canary/examples/with-react-helmet/pages/_document.js
+export default class extends Document {
+  static async getInitialProps(ctx) {
+    const originalRenderPage = ctx.renderPage;
+    try {
+      ctx.renderPage = () =>
+        originalRenderPage({
+          enhanceApp: App => props => <App {...props} />,
+        });
+
+      const documentProps = await Document.getInitialProps(ctx);
+      return {
+        ...documentProps,
+        helmet: Helmet.renderStatic(),
+        styles: <>{documentProps.styles}</>,
+      };
+    } finally {
+    }
+  }
+
+  get helmetHtmlAttrComponents() {
+    return this.props.helmet.htmlAttributes.toComponent();
+  }
+
+  get helmetBodyAttrComponents() {
+    return this.props.helmet.bodyAttributes.toComponent();
+  }
+
+  get helmetHeadComponents() {
+    return Object.keys(this.props.helmet)
+      .filter(el => el !== 'htmlAttributes' && el !== 'bodyAttributes')
+      .map(el => this.props.helmet[el].toComponent());
+  }
+
+  get helmetJsx() {
+    const title = 'Almond';
+    return (
+      <Helmet>
+        <title>{title}</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta property="og:title" content={title} />
+        <link
+          rel="stylesheet"
+          type="text/css"
+          href="https://rsms.me/inter/inter.css"
+        />
+      </Helmet>
+    );
+  }
+
+  render() {
```

Comment:
How does this fit in the existing layout?

Reply:
Will have to make some modifications to work with existing express